CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -39,15 +39,15 @@ internal static class JsonFileDalUtils
         /// <summary>
         /// A remark to be included in the exchange header file.
         /// </summary>
-        public const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
+        public const string DefaultExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
```

Comment:
Would it be possible to revert this to the `development` version (`private` + old name) and change:
```
        public void UpdateExchangeFileHeader(Person person, string headerCopyright = JsonFileDalUtils.DefaultExchangeHeaderCopyright, string headerRemark = JsonFileDalUtils.DefaultExchangeHeaderRemark)
        {
            var exchangeFileHeader = JsonFileDalUtils.CreateExchangeFileHeader(person);
            exchangeFileHeader.Remark = headerRemark;
            exchangeFileHeader.Copyright = headerCopyright;

            this.FileHeader = exchangeFileHeader;
        }
```
to something like
```
        public void UpdateExchangeFileHeader(Person person, string headerCopyright, string headerRemark)
        {
            var exchangeFileHeader = JsonFileDalUtils.CreateExchangeFileHeader(person);
            exchangeFileHeader.Remark = headerRemark ?? exchangeFileHeader.Remark;
            exchangeFileHeader.Copyright = headerCopyright ?? exchangeFileHeader.Copyright;

            this.FileHeader = exchangeFileHeader;
        }
```

Seems a lot easier to follow.